tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,18 +14,18 @@
 class Homescreen(Base):
 
     name = 'Default Home Screen'
-    manifest_url = '{}verticalhome{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)
```

Comment:
Are you using an older tree, I removed this already in my last pull request regarding this, I thought.